packages/tools/pluggable-widgets-tools/configs/rollup.config.js

```diff
@@ -265,38 +277,15 @@ const imagesAndFonts = [
     "**/*.woff(2)?",
     "**/*.eot"
 ];
-const nativeExtensions = [".native.js", ".js", ".jsx", ".ts", ".tsx"];
+
 const nativeExternal = [
     /^mendix\//,
-    "@react-native-community/art",
-    "@react-native-community/async-storage",
-    "@react-native-community/cameraroll",
-    "@react-native-community/geolocation",
-    "@react-native-community/netinfo",
-    "@react-native-firebase/analytics",
-    "@react-native-firebase/app",
-    "@react-native-firebase/crashlytics",
-    "@react-native-firebase/messaging",
-    "@react-native-firebase/ml-vision",
+    /^react-native(\/|$)/,
     "big.js",
     "react",
-    "react-native",
-    "react-native-camera",
-    "react-native-device-info",
-    "react-native-firebase",
-    "react-native-geocoder",
     /react-native-gesture-handler\/*/,
-    "react-native-image-picker",
-    "react-native-inappbrowser-reborn",
-    "react-native-localize",
-    "react-native-maps",
-    "react-native-reanimated",
-    "react-native-sound",
-    "react-native-svg",
-    "react-native-touch-id",
-    "react-native-vector-icons",
-    "react-native-video",
-    "react-native-view-shot",
-    "react-native-webview",
-    "react-navigation"
+    /^react-native-reanimated(\/|$)/,
+    /^react-native-svg(\/|$)/,
+    /^react-native-vector-icons(\/|$)/,
+    /^react-navigatio(\/|$)/
```

Comment:
There's a typo here, should be `react-navigation`. And do we need the same fix for the rollup config for JS actions?

Reply:
Done